chore/add-react-fs-structure (#17)
yeti/best-practices: docs/react-fs-structure.md

### Services

The `Services` folder is reserved for JS classes that provide utility in a thematically similar way. An example would be an HTTP service that handles all types of HTTP requests within the app.

### Shared

The `Shared` folder is intended for anything that will be consistently used across the app in more than one module. Two examples are the `Components` and `Types` directories. The `Components` directory will include components that will be consistently used across the app such as a `Text`, `Button` or `Modal` component. The `Types` directory will include any types that may be used across the app. This can include both `interfaces` as well as `enums`. If you find that there is anything else that you are using consistently across the app, it may make sense to include those here. Some examples might be, `StyledComponents`, `Utils`, or `Styles`.

### Static

The `Static` folder is intended for static files such as:

- Fonts
- Images
- SVGS
- Videos

## Tests

> While we currently don't have an established pattern for how we would like to write tests, we are currently suggesting that you colocate test files next to the file they're testing (e.g. utils.test.ts living right next to utils.ts).

Reviewer: ```suggestion
While we currently don't have an established pattern for how we would like to write tests, our current suggestion is to colocate test files next to the file they're testing (e.g. utils.test.ts living right next to utils.ts).
```
